Header header

float32 pm_10     # PM 10 concentration (ug/m^3)
float32 pm_25     # PM 2.5 concentration (ug/m^3)
int32 sensor_id
